### PR TITLE
MGDOBR-164: kustomize project uses manager and not the runner

### PR DESCRIPTION
**Pull request by r00ta**
[MGDOBR-164](https://issues.redhat.com/browse/MGDOBR-164) 

The goal of this PR is to deploy the manager application and not the runner. The `runner` module is not deleted in this PR, because it still contains the disabled e2e tests. 

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [x] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: manager/src/main/docker/Dockerfile.jvm [hunk 1]
```diff
@@ -1,55 +0,0 @@
-####
```
Comment (r00ta) [inline manager/src/main/docker/Dockerfile.jvm:1]: This was an initial leftover, we have a `docker` folder in the root for the dockerfiles of the applications

File: use-cases/ansible-tower-integration/README.md [hunk 2]
```diff
@@ -268,10 +263,11 @@ mvn clean compile quarkus:dev -f use-cases/ansible-tower-integration/ansible-gat
 
 ## Run
 
-To run the demo, simply send events matching the processor to the bridge endpoint. These are test events made only for this demo. This is the request:
+To run the demo, simply send events matching the processor to the bridge endpoint. These are test events made only for this demo. This is the request (change the endpoint 
+according to the Bridge Ingress endpoint you have deployed):
 
 ```bash
-curl --request POST 'http://localhost:8080/ingress/events/724439ac-a2ef-45e9-8b97-e6812afc0911' \
+curl --request POST 'http://<your_bridge_ingress_endpoint>/events' \
```
Comment (r00ta) [inline use-cases/ansible-tower-integration/README.md:270]: @kostola fyi

File: kustomize/overlays/prod/kustomization.yaml [hunk 3]
```diff
@@ -1,18 +1,19 @@
 images:
-- name: event-bridge-all-in-one
-  newName: quay.io/5733d9e2be6485d52ffa08870cabdee0/event-bridge-all-in-one
-  newTag: ab0f94c845368675f8fa01c49b3a4150d747b91e-jvm
+<<<<<<< HEAD
```
Comment (sutaakar) [inline kustomize/overlays/prod/kustomization.yaml:2]: Rebasing leftover?

**Review by sutaakar — APPROVED** [review]
Looks good to me

File: kustomize/overlays/prod/kustomization.yaml (hunk 3 above)
Comment (r00ta) [reply kustomize/overlays/prod/kustomization.yaml]: wops! Thank you :)

**Review by robpblake — APPROVED** [review]

**Pull request merged**